guides/source/action_mailbox_basics.md

```diff
@@ -18,30 +18,41 @@ After reading this guide, you will know:
 What is Action Mailbox?
 -----------------------
 
-Action Mailbox routes incoming emails to controller-like mailboxes for
-processing in Rails. It ships with ingresses for Mailgun, Mandrill, Postmark,
-and SendGrid. You can also handle inbound mails directly via the built-in Exim,
-Postfix, and Qmail ingresses.
+Action Mailbox routes incoming emails to controller-like mailboxes for processing in your Rails application. Action Mailbox is for receiving email, while [Action Mailer](action_mailer_basics.html) is for *sending* them.
```

Comment:
I really like how this streamlines the introduction to Action Mailbox and clarifying the difference to Action Mailer! 

Something I think could have tripped up developers in the past.

Reply:
Yes exactly! Since they both have to do with emails.